### ISSUE TITLE
Camera live stream entity not working when top banner (toolbar) is disabled

### ISSUE THREAD

**Issue by erich74** [j-a-n/lovelace-wallpanel#238]
I have a couple of dashboards with a camera live stream entity (Go2RTS) / RSTP .
I have noticed that whenever the header (top toolbar) is disabled, the stream is not working. As soon as I re enable the header, camera live stream is working fine again.
This is very strange because I have also tested with the Kiosk Mode HACS (https://github.com/NemesisRE/kiosk-mode) and this is the exact same issue. Side panel for instance does not affect the live stream, only the top banner.

**Versions:**
 - Home Assistant: 2024.6.3
 - WallPanel 4.25.2

**Comment by j-a-n** [comment]
Which card type are you using for the live stream?
Which browser are you using?
Are there any error messages in the browser console?

**Comment by j-a-n** [comment]
Just tested with custom:webrtc-camera and Google Chrome.
Everything is working fine.
Please try if it is working with Google Chrome or Firefox.

**Comment by erich74** [comment]
I did try with Chrome and the home assistant app and this is the same issue. 
Maybe my tablet which is not able to cope with the video stream?

**Comment by Pscharumbel** [comment]
Hi! I have the same strange problem. I am using Wallpanel on an LG TV. Recently, I got a Tapo C211 camera. Unfortunately, I can't use the live mode when I have fullscreen: true set. In that case, it doesn't refresh the image and keeps exiting the screensaver. However, the camera's auto mode works. If I set fullscreen to false (I then have a gray bar at the top), the live mode works. However, the image quality of the camera is very pixelated.
